test/python/algorithms/test_skip_qobj_validation.py

```diff
@@ -109,7 +110,21 @@ def test_w_noise(self):
             from qiskit.providers.aer.noise import NoiseModel
             from qiskit import Aer
 
-            self.backend = Aer.get_backend("qasm_simulator")
+            msg = (
+                "The method ``qiskit.circuit.quantumcircuit.QuantumCircuit.i()`` is deprecated as "
+                + "of qiskit-terra 0.45.0. It will be removed no earlier than 3 months after the "
+                + "release date. Use QuantumCircuit.id as direct replacement."
+            )
+
+            with warnings.catch_warnings():
+                warnings.filterwarnings(
+                    "default",
+                    message=msg,
+                    category=DeprecationWarning,
+                    module="test.python.algorithms.test_skip_qobj_validation",
```

Comment:
I just double checked how I did it in #10792, and realized the module needs to be specified differently:
```suggestion
                    module=r"test\.python\.algorithms\.test_skip_qobj_validation",
```
I would also argue that there is probably no need to give it the message, unless there is any other deprecation warning that you want to make sure `get_backend` is not filtering (which I guess not).